hil/cli.py

```diff
@@ -758,8 +758,7 @@ def list_headnode_images():
 @cmd
 def show_console(node):
     """Display console log for <node>"""
-    url = object_url('node', node, 'console')
-    do_get(url)
+    print C.node.show_console(node)
```

Comment:
Let's start putting parens around our print statements; this is a python 2 vs. 3 compat thing.

Reply:
sure, and Naved has to modify this in the new cli anyways I believe.